tests/testing_objects.py

```diff
@@ -65,31 +65,65 @@
+from crawler.rabbit.messages.parsers.create_plate_message import (
+    FIELD_MESSAGE_CREATE_DATE as CREATE_PLATE_MESSAGE_CREATE_DATE,
+)
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_MESSAGE_UUID as CREATE_PLATE_MESSAGE_UUID
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_MUST_SEQUENCE as CREATE_PLATE_MUST_SEQUENCE
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_PLATE as CREATE_PLATE_PLATE
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_PLATE_BARCODE as CREATE_PLATE_PLATE_BARCODE
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_PLATE_COORDINATE as CREATE_PLATE_PLATE_COORDINATE
+from crawler.rabbit.messages.parsers.create_plate_message import (
     FIELD_PREFERENTIALLY_SEQUENCE as CREATE_PLATE_PREFERENTIALLY_SEQUENCE,
 )
-from crawler.rabbit.messages.create_plate_message import FIELD_RESULT as CREATE_PLATE_RESULT
-from crawler.rabbit.messages.create_plate_message import FIELD_RNA_ID as CREATE_PLATE_RNA_ID
-from crawler.rabbit.messages.create_plate_message import FIELD_ROOT_SAMPLE_ID as CREATE_PLATE_ROOT_SAMPLE_ID
-from crawler.rabbit.messages.create_plate_message import FIELD_SAMPLE_UUID as CREATE_PLATE_SAMPLE_UUID
-from crawler.rabbit.messages.create_plate_message import FIELD_SAMPLES as CREATE_PLATE_SAMPLES
-from crawler.rabbit.messages.create_plate_message import FIELD_TESTED_DATE as CREATE_PLATE_TESTED_DATE
-from crawler.rabbit.messages.update_sample_message import FIELD_MESSAGE_CREATE_DATE as UPDATE_SAMPLE_MESSAGE_CREATE_DATE
-from crawler.rabbit.messages.update_sample_message import FIELD_MESSAGE_UUID as UPDATE_SAMPLE_MESSAGE_UUID
-from crawler.rabbit.messages.update_sample_message import FIELD_NAME as UPDATE_SAMPLE_NAME
-from crawler.rabbit.messages.update_sample_message import FIELD_SAMPLE as UPDATE_SAMPLE_SAMPLE
-from crawler.rabbit.messages.update_sample_message import FIELD_SAMPLE_UUID as UPDATE_SAMPLE_SAMPLE_UUID
-from crawler.rabbit.messages.update_sample_message import FIELD_UPDATED_FIELDS as UPDATE_SAMPLE_UPDATED_FIELDS
-from crawler.rabbit.messages.update_sample_message import FIELD_VALUE as UPDATE_SAMPLE_VALUE
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_RESULT as CREATE_PLATE_RESULT
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_RNA_ID as CREATE_PLATE_RNA_ID
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_ROOT_SAMPLE_ID as CREATE_PLATE_ROOT_SAMPLE_ID
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_SAMPLE_UUID as CREATE_PLATE_SAMPLE_UUID
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_SAMPLES as CREATE_PLATE_SAMPLES
+from crawler.rabbit.messages.parsers.create_plate_message import FIELD_TESTED_DATE as CREATE_PLATE_TESTED_DATE
+from crawler.rabbit.messages.parsers.update_sample_message import (
+    FIELD_MESSAGE_CREATE_DATE as UPDATE_SAMPLE_MESSAGE_CREATE_DATE,
+)
```

Comment:
Why we rename the fields? The name they had was not right? Or if not, should this alias be defined inside the module we are importing from?

Reply:
The names clash in the namespace when we're creating three different types of message for testing.  The module shouldn't need to attempt to have uniquely named exports.  It is its own namespace.  When we import them into a new namespace it is our duty to ensure they don't clash.  As some do clash and need qualifiers, I've applied the qualifiers across the board to clarify when creating the test objects that we are using the correct keys.

Reply:
Ok